Makefile

```diff
@@ -223,3 +223,6 @@ dummy:
 	$(SPHINXBUILD) -b dummy $(ALLSPHINXOPTS) $(BUILDDIR)/dummy
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
+
+livehtml:
```

Comment:
Can you add it to the [`make help`](https://github.com/tocco/tocco-docs/blob/50b150d8156c79b122b6fc0c1521433a1c5b2574/Makefile#L18) output too.